src/en/pin/pin-8.md

```diff
@@ -1,11 +1,13 @@
-This pin doubles up as the UART transmit pin, TX. It's also commonly known as "Serial" and, by default, will output a Console from your Pi that, with a suitable Serial cable, you can use to control your Pi via the command-line.
+This pin doubles up as the UART transmit pin, TX. It's also commonly known as "Serial" and, by default, will output a Console from your Pi that, with a [suitable Serial cable](https://elinux.org/RPi_Serial_Connection), you can use to control your Pi via the command line.
 
-The UART pins are useful for setting up a "headless" Pi (a Pi without a screen) and getting it connected to a network.
+The UART pins can be useful for setting up a "headless" Pi (a Pi without a screen) and getting it connected to a network.
 
 UART can be used to talk to Serial GPS modules or sensors such as the PM5003, but you must make sure you disable the Serial Console in raspi-config first.
 
-On the Pi 3 and 4 the UART is, by default, used for Bluetooth and you may need to add "dtoverlay=miniuart-bt" to "/boot/config.txt" to achieve a stable.
+On the Pi 3 and 4 the UART is, by default, used for Bluetooth and you may need to add "dtoverlay=miniuart-bt" to "/boot/config.txt" to be able to use UART (although a reduced version, mini UART).
```

Comment:
Also the Pi Zero W

Reply:
Ah the pitfall of mentioning any specific Pi... the list is getting too darn long!